src/main/java/culturemedia/controller/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturoteca.controllers;
+
+import java.util.*;
+
+import culturoteca.exception.VideoNotFoundException;
+import culturoteca.model.Video;
+import culturoteca.service.impl.CultureMediaServiceImpl;
```

Comment:
is : import culturemedia.service.impl.CultureMediaService;
the CultureMediaService interface is imported, not the implementation